packages/amplify-category-auth/src/index.js

```diff
@@ -525,13 +526,16 @@ async function isSMSWorkflowEnabled(context, resourceName) {
  * Execute auth Push command with force yes
  * @param {Object} context - The amplify context.
  */
-const authPush = async context => {
-  const commandPath = path.normalize(path.join(__dirname, 'commands', category, 'push'));
-  const commandModule = await import(commandPath);
-  context.exeInfo = (context.exeInfo) || {};
-  context.exeInfo.inputParams = (context.exeInfo.inputParams) || {};
-  context.exeInfo.inputParams.yes = true; // force yes to avoid prompts
-  await commandModule.run(context);
+const authPushYes = async context => {
+  const exeInfoClone = Object.assign({}, (context.exeInfo) || {});
```

Comment:
nit: I think `{...context?.exeInfo}` does the same thing and is a little more concise

Reply:
updated